src/main.tsx

```diff
@@ -149,8 +138,22 @@ i18n
     },
   });
 
+// Initilization done after the App config has loaded:
+function onConfigLoaded() {
+  Sentry.init({
```

Comment:
Is this async? Should we also await this? Could we put it into `App.tsx` since most of the init code is already there?